README.adoc

```diff
@@ -192,8 +192,9 @@ To set up the cache:
 . On non-NixOS, edit `/etc/nix/nix.conf` and add the following lines:
 +
 ----
-substituters        = https://hydra.iohk.io https://iohk.cachix.org https://cache.nixos.org/
-trusted-public-keys = hydra.iohk.io:f/Ea+s+dFdN+3Y/G+FDgSq+a5NEWhJGzdjvKNGv0/EQ= iohk.cachix.org-1:DpRUyj7h7V830dp/i6Nti+NEO2/nhblbov/8MW7Rqoo= cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
+substituters = https://hydra.iohk.io https://cache.nixos.org/
```

Comment:
We want to add `https://cache.iog.io` here too

Reply:
In that case, it's missing in the https://github.com/input-output-hk/plutus/blob/master/CONTRIBUTING.adoc#installing-and-setting-up-nix repo. I'll add it there as well.

Reply:
Should we remove `https://cache.nixos.org`? Or do we need to add it in the NixOS config?

Reply:
That's the default cache. I don't know if it's used implicitly by NixOS if no cache is defined. I'd keep it, it doesn't hurt.